include/api/wasmedge/wasmedge.h

```diff
@@ -2701,6 +2730,71 @@ WASMEDGE_CAPI_EXPORT void WasmEdge_AsyncDelete(WasmEdge_Async *Cxt);
 
 // <<<<<<<< WasmEdge Async functions <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
 
+// >>>>>>>> WasmEdge Coroutine functions >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+
+/// Start to execute a WasmEdge_Coroutine execution.
+///
+/// \param Cxt the WasmEdge_Coroutine.
+WASMEDGE_CAPI_EXPORT void WasmEdge_CoroutineSpawn(WasmEdge_Coroutine *Cxt);
```

Comment:
Are you going to be creating a separate stack here (or is this a stackful or stackless coroutine)? If so it's useful to add the amount of memory used here.

Reply:
I am going to have a stackful coroutine. Would you like any other information (memory usage sounds useful)?